apps/web/public/sw.js

```diff
@@ -0,0 +1,411 @@
+/* Service Worker */
+
+const version = 1;
+const dbVersion = 2;
+const database = "inbox-zero";
+let DB = null;
+let dynamicName = `dynamicCache`;
+let IMAGE_CACHE = "IMAGE_CACHE";
+const LABEL_STORE = {
+  name: "labels",
+  key: "id",
+  indexes: [],
+};
+const EMAIL_STORE = {
+  name: "emails",
+  key: "gmailMessageId",
+  indexes: [{ name: "timestampIDX", property: "timestamp", params: {} }],
```

Comment:
not sure about naming conventions here, but what about just calling it `timestamp`? Is IDX a convention?